test/integration/api-conversion-v1beta1/01-assert.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: lifecycle.keptn.sh/v1alpha3
```

Comment:
to avoid naming collisions, please move the tests for converting from v1alpha -> v1beta1, and for v1beta1-> v1alpha into separate directories

Reply:
done